formatting.md

```diff
@@ -0,0 +1,103 @@
+# Formatting
```

Comment:
Shall we move this section to blocks.md either below or above the Escaping Text section?

Reply:
Yeah i think that makes sense with regard to un-scattering the documentation

Reply:
OK, sure.